docs/testing/sideload-office-add-ins-for-testing.md

```diff
@@ -1,57 +1,62 @@
+When you sideload an add-in, you're able to install the add-in without first putting it in an add-in catalog. This is useful when testing and developing your add-in because you can see how your add-in will appear and function.
 
-When you sideload an add-in, the add-in's manifest is stored in the browser's local storage, so if you clear the browser's cache, or switch to a different browser, you have to sideload the add-in again.
+When you sideload an add-in on the web, the add-in's manifest is stored in the browser's local storage, so if you clear the browser's cache, or switch to a different browser, you have to sideload the add-in again.
 
-Sideloading varies between host applications (for example, Excel).
+The steps to sideload an add-in on the web vary based on these factors:
 
-> [!NOTE]
-> Sideloading as described in this article is supported on Excel, OneNote, PowerPoint, and Word. To sideload an Outlook add-in, see [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md).
+- The host application (for example, Excel, Word, Outlook).
+- What tool created the add-in project (for example, Visual Studio, Yeoman generator for Office Add-ins, or neither)
+- Whether you are sideloading to retail Office on the web or to a Microsoft 365 tenancy.
 
-## Sideload an Office Add-in in Office on the web
+Go to the section or article that matches your scenario:
 
-This process is supported for **Excel**, **OneNote**, **PowerPoint**, and **Word** only. For other host applications, see the manual sideloading instructions in the following section. This example project assumes that you are using a project created with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).
+- If you are sideloading an Outlook Add-in, see the article [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md). *The remaining bullets apply to non-Outlook add-ins.*
+- If the add-in was created with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md), see [Sideload a Yeoman-created add-in to Office on the web](#sideload-a-yeoman-created-add-in-to-office-on-the-web).
+- If the add-in was created with Visual Studio, see [Sideload an add-in on the web when using Visual Studio](#sideload-an-add-in-on-the-web-when-using-visual-studio).
+- For all other cases, see one of these sections:
 
-1. Open [Office on the web](https://office.live.com/). Using the **Create** option, make a document in **Excel**, **OneNote**, **PowerPoint**, or **Word**. In this new document, select **Share** in the ribbon, select **Copy Link**, and copy the URL.
+  - If you are sideloading to retail Office on the web, see [Manually sideload an add-in to Office on the web](#manually-sideload-an-add-in-to-office-on-the-web).
```

Comment:
Is there a different option for VL?

Reply:
I don't understand the question. What is VL?

Reply:
Sorry - I mean volume-licensed

Reply:
VL is desktop. There are other articles that cover sideloading on desktop. This aritcle is only about sideloading on the web.

Reply:
We clarified this off line. I changed the terminology to clarify that the distinction is between M365 account and Microsoft Account.